### ISSUE TITLE
Brush Configuration Stuck

### ISSUE THREAD

**Issue by 1itt1eB0y** [flxzt/rnote#206]
Click `Brush Configuration`, it will stuck there, can't be closed.
![图片](https://user-images.githubusercontent.com/29831189/173736020-ff8f8526-3112-4acb-9851-3805c6df1352.png)

SMP PREEMPT Debian 5.16.14-1kali2 (2022-03-23)
Rnote 0.5.3

**Comment by flxzt** [comment]
is this happening after you modified the "pressure curve" or "stroke dots distribution" preference? If yes, then [it is a GTK4 issue](https://gitlab.gnome.org/GNOME/gtk/-/issues/4529)

**Comment by 1itt1eB0y** [comment]
> is this happening after you modified the "pressure curve" or "stroke dots distribution" preference? If yes, then [it is a GTK4 issue](https://gitlab.gnome.org/GNOME/gtk/-/issues/4529)

Yep, I met this problem when I clicked the "stroke dots distribution".

**Comment by TeiVDK** [comment]
I can confirm this being the case after changing the pressure curve. As mentioned above, this seems to be a GTK4 issue.

**Comment by flxzt** [comment]
There is a workaround: After selecting the desired value from the the child popover, you need to open it again. While it is opened, you can then close both by clicking somewhere outside of them

**Comment by bevinhex** [comment]
workaround is not working for me, using 0.6.0,  Even if it were gtk4 issue, it should not take much effort to add a close button there so that people can actually click it.

**Comment by flxzt** [comment]
you're right, a small (x) button to close the popover could be added quite easily. It seems that nobody from the Gtk dev team really cares about this issue / they don't have the ressources to fix it.

**Comment by flxzt** [comment]
Close buttons for all configuration popovers are now added to work around the issue.

**Comment by Doublonmousse** [comment]
Hopeful this https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/9050 is a correct solution (so that it gets approved and merged in the gtk 4.22 release).

We'll see

**Comment by Doublonmousse** [comment]
The upstream issue (https://gitlab.gnome.org/GNOME/gtk/-/work_items/4529) was closed, with the fix to be included in gtk 4.24
